### PR TITLE
New trip

### PR DESCRIPTION
**Pull request by adikakade31**
1. Shifted Logout and Delete Account to navigation drawer
2. Added CreateStoryActivity and StoryActivity with static places in story.

### CODE REVIEW

**Comment by adikakade31** [comment]
Model -> StoryPlace.java is used just for now as we are using static places. This class will be deleted  once we implement api fetch to save in parse.

File: app/src/main/java/com/codepath/travel/activities/HomeActivity.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+import com.codepath.travel.models.User;
+import com.parse.ParseUser;
+import com.parse.ui.ParseLoginBuilder;
+import com.facebook.FacebookSdk;
+import com.facebook.appevents.AppEventsLogger;
+import com.parse.ParseFacebookUtils;
+
+public class HomeActivity extends AppCompatActivity {
+    private static final String TAG = HomeActivity.class.getSimpleName();
+    private static final int LOGIN_REQUEST = 0;
```
Comment (prav90) [inline app/src/main/java/com/codepath/travel/activities/HomeActivity.java]: can we move this request_values to a separate file/enum?

File: app/src/main/java/com/codepath/travel/activities/SearchActivity.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.codepath.travel.R;
+
+@SuppressWarnings("deprecated")
```
Comment (heyhuyen) [inline app/src/main/java/com/codepath/travel/activities/SearchActivity.java:11]: What's throwing the deprecated warning?

Comment (adikakade31) [reply app/src/main/java/com/codepath/travel/activities/SearchActivity.java:11]: This class is not used for now. You can ignore it

**Review by prav90 — APPROVED** [review]
Looks good. Let's try to create reusable fragments going forward and save us a lot of refactoring or redoing the same thing over again later. It's better to start thinking in that way earlier :)

File: app/src/main/res/layout/activity_create_story.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        layout="@layout/toolbar"
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize" />
+
+    <!--<android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        app:popupTheme="@style/AppTheme.PopupOverlay"/>-->
```
Comment (heyhuyen) [inline app/src/main/res/layout/activity_create_story.xml:25]: We can delete this right? and in the other activity layouts as well.

Comment (adikakade31) [reply app/src/main/res/layout/activity_create_story.xml:25]: yup

**Comment by heyhuyen** [comment]
Looks good! Let's try squashing commits and then merging? Going forward, I'm okay with self-merging if others have given the OK.

**Comment by adikakade31** [comment]
Let me know if there are no changes, I can merge the pull request :)

**Comment by heyhuyen** [comment]
Go for it!

**Pull request merged**